code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -568,3 +568,24 @@ var/global/list/plantbag_colour_choices = list("plantbag", "green red stripe", "
 	storage_slots = 3
 	w_class = W_CLASS_LARGE
 	slot_flags = SLOT_BELT | SLOT_POCKET
+
+
+
+// -----------------------------
+//          Xenobiology Bag
+// -----------------------------
+
+
+/obj/item/weapon/storage/bag/xenobio
+	icon = 'icons/obj/storage/storage.dmi'
+	icon_state = "slimebag"
+	name = "Extract Bag"
+	desc = "A bag designed to hold slime extract and other slime-related products."
+	item_state = "satchel"
+	origin_tech = Tc_BLUESPACE + "=2" + Tc_MATERIALS + "=2" + Tc_ENGINEERING + "=2"
```

Comment:
I think this is not formatted correctly, needs some `;`s in there, look at other implementations.